### ISSUE TITLE
Did Coinbase change and break the posting of keybase.io proofs?

### ISSUE THREAD

**Issue by PublicStartup** [keybase/keybase-issues#1937]
Followed instructions precisely. I am not clueless.

Coinbase must have changed something, or is malfunctioning.

There are two ways to get to a "settings" page for my Coinbase account:

1. https://www.coinbase.com/settings

2. https://www.coinbase.com/jasoncoombsceo#settings

#2 is the method instructed by keybase.io -- visiting URL #2 above while logged in to my Coinbase account gives me a "Public Key" text field and a Save button. No matter what I type or paste into the text field, clicking Save appears to succeed (the text is there upon refresh or closing the browser and logging in to my Coinbase account over again) but the text is not visible when visiting the URL using an incognito browser session or when not logged-in to Coinbase.

Visiting https://www.coinbase.com/jasoncoombsceo displays only my name, location and Caption fields. These fields can be set using URL #1 above, but Caption is limited to 200 characters.

For the reasons discussed above, I was unable to complete the Coinbase proof steps. I believe it would have worked if Coinbase had displayed my proof text at URL #2 above after I pasted it into the "Public Key" text field and clicked Save. Presumably this is how keybase.io presumes Coinbase will function, but it is not functioning.

**Comment by malgorithms** [comment]
Hi @PublicStartup  - I see your post here:

https://www.coinbase.com/jasoncoombsceo/public-key

which is where Keybase looks for proofs, although that doesn't look like a normal Keybase proof.

**Comment by PublicStartup** [comment]
Thank you, I could not see my public-key on my coinbase profile after adding it and did not know about the public-key URL. The keybase proof failed to verify because I had not manually removed the CRLF's that were added when I copied and pasted from the Windows command prompt.

Proof succeeded this time. The proof verification code should ignore whitespace, but this is a known issue. The user experience of posting a proof to coinbase needs to be improved. Ideally coinbase would help with this by adding another user data field so that keybase doesn't need to temporarily use the coinbase "public-key" user data for this purpose. At the very least users should be told to look for the proof they are trying to post at their coinbase.com/user/public-key URL if they have any problems with proof verification.
